File: lab№5_2.cpp
#include <iostream>
#include <cmath>
using namespace std;
int main()
{
    double xA, yA;
    double xB, yB;
    double xC, yC;
    cout << "Enter A: ";
    cin >> xA >> yA;
    cout << "Enter B: ";
    cin >> xB >> yB;
    cout << "Enter C: ";
    cin >> xC >> yC;
    double A = sqrt(xA * xA + yA * yA);
    double B = sqrt(xB * xB + yB * yB);
    double C = sqrt(xC * xC + yC * yC);
    if (A < B && A < C)
    {
        cout << "Nearest point: A";
    }
    else if (B < A && B < C)
    {
        cout << "Nearest point: B";
    }
    else
    {
        cout << "Nearest point: C";
    }
    return 0;
}

File: lab№5_2.py
import math
xa = float(input("Введіть xA: "))
ya = float(input("Введіть yA: "))
xb = float(input("Введіть xB: "))
yb = float(input("Введіть yB: "))
xc = float(input("Введіть xC: "))
yc = float(input("Введіть yC: "))
A = math.sqrt(xa ** 2 + ya ** 2)
B = math.sqrt(xb ** 2 + yb ** 2)
C = math.sqrt(xc ** 2 + yc ** 2)
if A > B and A > C:
    print("Найбільша відстань у точки A")
elif B > A and B > C:
    print("Найбільша відстань у точки B")
else:
    print("Найбільша відстань у точки C")